$MirOS: ports/devel/soup/patches/patch-configure_in,v 1.2 2005/07/24 16:53:13 bsiegert Exp $
--- configure.in.orig	Thu Jan  2 20:03:39 2003
+++ configure.in	Tue Dec 27 23:09:35 2005
@@ -424,15 +424,20 @@ enable_apache="no"
 
 AC_PATH_PROG(APXS,apxs,no)
 if test x$APXS != xno; then
-	APACHE_CFLAGS="-I`apxs -q INCLUDEDIR CFLAGS`"
-	APACHE_LIBS=`apxs -q LDFLAGS_SHLIB LIBS_SHLIB`
+	APACHE_CFLAGS="-I`apxs -q INCLUDEDIR CFLAGS` -Wno-error"
+	APACHE_LIBS=`apxs -q LDFLAGS_SHLIB LIBS_SHLIB | sed "s/-Wl,-bind_at_load/-bind_at_load/"`
 
 	save_CPPFLAGS=$CPPFLAGS
 	CPPFLAGS=$APACHE_CFLAGS
+	AC_CHECK_HEADERS(ap_config.h)
 	AC_CHECK_HEADERS(httpd.h http_config.h http_core.h http_log.h http_main.h http_protocol.h,
 			  [enable_apache="yes"],
 			  [enable_apache="no"
-			   break])
+			   break],
+			  [#if HAVE_AP_CONFIG_H
+			   #include <ap_config.h>
+			   #endif
+			  ])
 	CPPFLAGS=$save_CPPFLAGS
 
 	AC_SUBST(APACHE_CFLAGS)
